cds_migrator_kit/records/utils.py

```diff
@@ -60,3 +64,77 @@ def update_access(data, *access):
             current_rules[k] = list(current_x_rules)
 
     data['_access'] = current_rules
+
+
+def prepare_serials(serial, logger, rectype, item):
+    """Prepare serials records to migrate."""
+    return split_serials(serial, logger, rectype, item)
+
+
+def check_for_duplicated_serials(serial):
+    """Check if the serial already exists."""
+    _logs_path = current_app.config['CDS_MIGRATOR_KIT_LOGS_PATH']
+    filepath = os.path.join(_logs_path, 'serials.json')
+    with open(filepath, 'r+') as file:
+        all_serials = json.load(file)
+        for stored_serial in all_serials:
+            ratio = fuzz.ratio(serial['title']['title'],
+                               stored_serial['title']['title'])
+            if serial['title']['title'] == stored_serial['title']['title'] or \
+                ('issn' in serial and serial['issn'] is not None and
+                    serial['issn'] == stored_serial['issn']):
+                return stored_serial, 100
+
+            elif 95 <= ratio < 100:
```

Comment:
for the future, maybe add this `ratio` value in a config in case you need to adjust in the future, because the data will be cleaner and cleaner?

Reply:
sure I was planning to do that in the next iteration